maps/OlivinePort.asm

```diff
@@ -82,12 +82,7 @@ OlivinePortWalkUpToShipScript:
 	opentext
 	checkevent EVENT_FAST_SHIP_FIRST_TIME
 	iffalse .FirstTime
-	readvar VAR_WEEKDAY
-	ifequal SUNDAY, .NextShipMonday
```

Comment:
For now I suggest keeping these in case its decided that this change should be locked behind a flag. I suspect that it probably won't need to be though, so once this gets properly merged into the main branch for speedchoice V7 we should propose to delete this.